backend/app/Http/Controllers/Api/CoachingApiController.php

```
@@ -0,0 +1,20 @@
+
+namespace App\Http\Controllers\Api;
+
+use App\Http\Controllers\Controller;
+use App\Models\Coach;
+
+class CoachingApiController extends Controller
+{
+
+    public function getCoaches()
+    {
+        return Coach::paginate();
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\Coach' in method 'getCoaches'. (Clean Code Rules)

(StaticAccess)